res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -452,9 +446,9 @@ Jockey3ME.MixerVol = function (channel, control, value, status, group) {
     currentDeck += 2;
   }
   if (control == 0x2D || control == 0x6C) {
-	  engine.setValue("[Channel" + currentDeck + "]","pregain",script.absoluteNonLin(value,0,1,4,0,127));
+	  engine.setValue("[Channel" + currentDeck + "]","pregain",script.absoluteLin(value,0,script.absoluteLin(value,0,4,0,127),0,127));
```

Comment:
It was but it has not handled the curve of how the setValue() acts like. In that way it has a curve more like a circular. That's what i mean that it is weirdly with setValue() instead of setParameter().